### PR TITLE
Do not use 'PROTOs' plural form in documentation

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Fix https://github.com/omichel/webots-doc/issues/158.
Related to #4188.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by DavidMansolino** [comment]
Just a small question, what is the rule for choosing between `PROTO nodes` and `PROTO models`?

**Comment by stefaniapedrazzi** [comment]
There is no rule, personally I choose the one that sounds better in the sentence context.

**Comment by DavidMansolino** [comment]
Ok thank you for the explanation.

:+1:

File: reference/slot.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Slot {
...
 node in the `endPoint` field of the second [Slot](#slot). Furthermore, the
 second [Slot](#slot) can be added only if it has the same `type` as the first
-one.  The [Slot](#slot) node is particularly usefull with PROTOs, it allows the
+one. The [Slot](#slot) node is particularly useful with PROTO nodes, it allows the
 user to constrain the type of nodes that can be added in an extension field of
 the PROTO. Imagine for example that you have an armed robot in which you can
 plug different kinds of hands. In order to do so you will put the hand as an
 extension field of your robot, you will then be able to add all the different
-PROTOs of hand that you have made. But nothing prevent you to add a PROTO of
+PROTO models of hand that you have made. But nothing prevents you from adding a PROTO of
```
Comment (omichel) [inline reference/slot.md]: PROTO &rarr; PROTO model

Comment (omichel) [inline reference/slot.md]: I assume that we should also use PROTO model or PROTO node at the singular form, right?

**Comment by stefaniapedrazzi** [comment]
Sorry, the page didn't update I didn't see the comments.
I will open a new branch.

File: reference/slot.md (hunk 1 above)
Comment (stefaniapedrazzi) [inline reference/slot.md]: I don't know.
I would say yes for consistency, but we are not doing it for node names ('Solid', 'LightSensor', etc.).

Comment (omichel) [inline reference/slot.md]: I don't know either... But to me, it is not very important.

**Pull request merged**